2022/08/02 23:42:50 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "giix". in C:\Yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('giix')
#1 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\projeto-yii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=giix
---
2022/08/02 23:44:32 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: could not find driver in C:\Yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\Yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\Yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\Yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\Yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Yii\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\Yii\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\Yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/index.php?r=gii/default/index
---
2022/08/03 11:22:47 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'db_blogconexa' in C:\Yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\Yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\Yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\Yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\Yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Yii\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\Yii\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\Yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/?r=gii
---
2022/08/03 11:23:46 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'db_blogconexa' in C:\Yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\Yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\Yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\Yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\Yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Yii\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\Yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\Yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixModel/i...')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/giixModel/index
HTTP_REFERER=http://localhost:3000/?r=gii
---
2022/08/03 11:23:57 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'db_blogconexa' in C:\Yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\Yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\Yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\Yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\Yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Yii\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\Yii\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\Yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost:3000/?r=gii
---
2022/08/03 11:28:07 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'mydb' in C:\Yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\Yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\Yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\Yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\Yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Yii\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\Yii\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\Yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/?r=gii
---
2022/08/03 11:34:30 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'mydb' in C:\Yii\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\Yii\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\Yii\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\Yii\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\Yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Yii\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\Yii\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\Yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/?r=gii
---
2022/08/03 13:21:55 [error] [exception.CException] CException: Property "CPhpMessageSource.extensionBasePaths" is not defined. in C:\Yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\Yii\framework\YiiBase.php(227): CComponent->__set('extensionBasePa...', Array)
#1 C:\Yii\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\Yii\framework\YiiBase.php(619): CModule->getComponent('messages')
#3 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\GiixCrudCode.php(103): YiiBase::t('app', 'password')
#4 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\templates\default\_form.php(29): GiixCrudCode->generateActiveField('Usuario', Object(CMysqlColumnSchema))
#5 C:\Yii\framework\gii\CCodeModel.php(321): require('C:\\projeto-yii\\...')
#6 C:\Yii\framework\gii\generators\crud\CrudCode.php(97): CCodeModel->render('C:\\projeto-yii\\...')
#7 C:\Yii\framework\gii\CCodeGenerator.php(162): CrudCode->prepare()
#8 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#9 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#10 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#14 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#15 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\projeto-yii\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/index.php?r=gii/giixCrud/index
---
2022/08/03 13:36:53 [error] [exception.CException] CException: Property "CPhpMessageSource.extensionBasePaths" is not defined. in C:\Yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\Yii\framework\YiiBase.php(227): CComponent->__set('extensionBasePa...', Array)
#1 C:\Yii\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\Yii\framework\YiiBase.php(619): CModule->getComponent('messages')
#3 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\GiixCrudCode.php(103): YiiBase::t('app', 'password')
#4 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\templates\default\_form.php(29): GiixCrudCode->generateActiveField('Usuario', Object(CMysqlColumnSchema))
#5 C:\Yii\framework\gii\CCodeModel.php(321): require('C:\\projeto-yii\\...')
#6 C:\Yii\framework\gii\generators\crud\CrudCode.php(97): CCodeModel->render('C:\\projeto-yii\\...')
#7 C:\Yii\framework\gii\CCodeGenerator.php(162): CrudCode->prepare()
#8 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#9 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#10 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#14 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#15 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\projeto-yii\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/index.php?r=gii/giixCrud/index
---
2022/08/03 13:37:21 [error] [exception.CException] CException: Property "CPhpMessageSource.extensionBasePaths" is not defined. in C:\Yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\Yii\framework\YiiBase.php(227): CComponent->__set('extensionBasePa...', Array)
#1 C:\Yii\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\Yii\framework\YiiBase.php(619): CModule->getComponent('messages')
#3 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\GiixCrudCode.php(103): YiiBase::t('app', 'password')
#4 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\templates\default\_form.php(29): GiixCrudCode->generateActiveField('Post', Object(CMysqlColumnSchema))
#5 C:\Yii\framework\gii\CCodeModel.php(321): require('C:\\projeto-yii\\...')
#6 C:\Yii\framework\gii\generators\crud\CrudCode.php(97): CCodeModel->render('C:\\projeto-yii\\...')
#7 C:\Yii\framework\gii\CCodeGenerator.php(162): CrudCode->prepare()
#8 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#9 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#10 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#14 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#15 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\projeto-yii\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/index.php?r=gii/giixCrud/index
---
2022/08/03 13:38:20 [error] [exception.CException] CException: Property "CPhpMessageSource.extensionBasePaths" is not defined. in C:\Yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\Yii\framework\YiiBase.php(227): CComponent->__set('extensionBasePa...', Array)
#1 C:\Yii\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\Yii\framework\YiiBase.php(619): CModule->getComponent('messages')
#3 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\GiixCrudCode.php(103): YiiBase::t('app', 'password')
#4 C:\projeto-yii\protected\extensions\giix\generators\giixCrud\templates\default\_form.php(29): GiixCrudCode->generateActiveField('Usuario', Object(CMysqlColumnSchema))
#5 C:\Yii\framework\gii\CCodeModel.php(321): require('C:\\projeto-yii\\...')
#6 C:\Yii\framework\gii\generators\crud\CrudCode.php(97): CCodeModel->render('C:\\projeto-yii\\...')
#7 C:\Yii\framework\gii\CCodeGenerator.php(162): CrudCode->prepare()
#8 C:\Yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#9 C:\Yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#10 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#14 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/giixCrud/in...')
#15 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\projeto-yii\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=gii/giixCrud/index
HTTP_REFERER=http://localhost:3000/index.php?r=gii/giixCrud/index
---
<<<<<<< HEAD
2022/08/04 08:15:04 [error] [php] Undefined variable: email (C:\projeto-yii\protected\components\UserIdentity.php:24)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (24)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (94)
2022/08/04 08:25:17 [error] [php] Undefined variable: email (C:\projeto-yii\protected\components\UserIdentity.php:24)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (24)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (94)
2022/08/04 08:27:41 [error] [php] Undefined variable: email (C:\projeto-yii\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (23)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (94)
2022/08/04 08:58:34 [error] [php] Undefined variable: email (C:\projeto-yii\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (23)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (94)
2022/08/04 08:59:24 [error] [php] Undefined variable: email (C:\projeto-yii\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (23)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (94)
2022/08/04 09:36:54 [error] [exception.CException] CException: Property "UserIdentity.email" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\projeto-yii\protected\components\UserIdentity.php(23): CComponent->__get('email')
#1 C:\projeto-yii\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#3 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#4 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\projeto-yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 09:40:57 [error] [exception.CException] CException: Property "UserIdentity.email" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\projeto-yii\protected\components\UserIdentity.php(23): CComponent->__get('email')
#1 C:\projeto-yii\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#3 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#4 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\projeto-yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 09:42:38 [error] [php] Undefined variable: user (C:\projeto-yii\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (23)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (94)
2022/08/04 09:46:37 [error] [php] Undefined variable: user (C:\projeto-yii\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(95): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (23)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (95)
2022/08/04 09:47:01 [error] [php] Undefined variable: user (C:\projeto-yii\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(95): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (23)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (95)
2022/08/04 09:47:56 [error] [php] Undefined variable: user (C:\projeto-yii\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\projeto-yii\protected\controllers\SiteController.php(95): LoginForm->validate()
#3 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\components\UserIdentity.php (23)
in C:\projeto-yii\protected\models\LoginForm.php (54)
in C:\projeto-yii\protected\controllers\SiteController.php (95)
2022/08/04 09:55:58 [error] [exception.CException] CException: Property "CDbCriteria.0" is not defined. in C:\Yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\Yii\framework\db\schema\CDbCriteria.php(160): CComponent->__set('0', 't@t.com')
#1 C:\Yii\framework\db\schema\CDbCommandBuilder.php(588): CDbCriteria->__construct(Array)
#2 C:\Yii\framework\db\ar\CActiveRecord.php(1463): CDbCommandBuilder->createCriteria(Array, Array)
#3 C:\projeto-yii\protected\components\UserIdentity.php(23): CActiveRecord->find(Array)
#4 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#5 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#6 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#7 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#8 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#9 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\projeto-yii\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 10:12:00 [error] [exception.CException] CException: Usuario and its behaviors do not have a method or closure named "validatePassword". in C:\Yii\framework\base\CComponent.php:266
Stack trace:
#0 C:\Yii\framework\db\ar\CActiveRecord.php(229): CComponent->__call('validatePasswor...', Array)
#1 C:\projeto-yii\protected\components\UserIdentity.php(28): CActiveRecord->__call('validatePasswor...', Array)
#2 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#3 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#4 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#5 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\projeto-yii\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 10:16:34 [error] [exception.CException] CException: Usuario and its behaviors do not have a method or closure named "validatePassword". in C:\Yii\framework\base\CComponent.php:266
Stack trace:
#0 C:\Yii\framework\db\ar\CActiveRecord.php(229): CComponent->__call('validatePasswor...', Array)
#1 C:\projeto-yii\protected\components\UserIdentity.php(26): CActiveRecord->__call('validatePasswor...', Array)
#2 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#3 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#4 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#5 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\projeto-yii\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 10:23:52 [error] [exception.Error] Error: Access to undeclared static property: Yii::$app in C:\projeto-yii\protected\components\UserIdentity.php:26
Stack trace:
#0 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#1 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#2 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#3 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#4 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#5 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#6 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#10 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#11 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\projeto-yii\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 10:51:11 [error] [exception.Error] Error: Using $this when not in object context in C:\projeto-yii\protected\models\Usuario.php:12
Stack trace:
#0 C:\projeto-yii\protected\components\UserIdentity.php(28): Usuario::validarSenha('4321')
#1 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#2 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#3 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#4 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\projeto-yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 10:52:03 [error] [php] Undefined variable: senha (C:\projeto-yii\protected\models\Usuario.php:12)
Stack trace:
#0 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#1 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#2 C:\Yii\framework\base\CModel.php(159): CInlineValidator->validate()
#3 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->validate()
#4 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\Yii\framework\web\CController.php(286): SiteController->runAction()
#7 C:\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 C:\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\projeto-yii\protected\models\Usuario.php (12)
in C:\projeto-yii\protected\components\UserIdentity.php (28)
in C:\projeto-yii\protected\models\LoginForm.php (55)
2022/08/04 11:22:28 [error] [exception.Error] Error: Using $this when not in object context in C:\projeto-yii\protected\models\Usuario.php:12
Stack trace:
#0 C:\projeto-yii\protected\components\UserIdentity.php(28): Usuario::validarSenha('4321')
#1 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#2 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#3 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#4 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\projeto-yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:26:24 [error] [exception.CException] CException: Property "Usuario.id" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\Yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\projeto-yii\protected\components\UserIdentity.php(37): CActiveRecord->__get('id')
#2 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#3 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#4 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#5 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\projeto-yii\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:27:37 [error] [exception.CException] CException: Property "Usuario.id" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\Yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\projeto-yii\protected\components\UserIdentity.php(37): CActiveRecord->__get('id')
#2 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#3 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#4 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#5 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\projeto-yii\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:28:36 [error] [exception.CException] CException: Property "UserIdentity._id" is not defined. in C:\Yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\projeto-yii\protected\components\UserIdentity.php(37): CComponent->__set('_id', '1')
#1 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#2 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#3 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#4 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\projeto-yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:33:50 [error] [exception.CException] CException: Property "Usuario.idUsuario" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\Yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('idUsuario')
#1 C:\projeto-yii\protected\components\UserIdentity.php(28): CActiveRecord->__get('idUsuario')
#2 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#3 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#4 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#5 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\projeto-yii\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:34:08 [error] [exception.CException] CException: Property "Usuario.idUsuario" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\Yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('idUsuario')
#1 C:\projeto-yii\protected\components\UserIdentity.php(28): CActiveRecord->__get('idUsuario')
#2 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#3 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#4 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#5 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\projeto-yii\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:35:32 [error] [exception.CException] CException: Property "UserIdentity._id" is not defined. in C:\Yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\projeto-yii\protected\components\UserIdentity.php(37): CComponent->__set('_id', '1')
#1 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#2 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#3 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#4 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#5 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#6 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#11 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#12 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\projeto-yii\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:37:26 [error] [exception.CException] CException: Property "Usuario.username" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\Yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('username')
#1 C:\projeto-yii\protected\components\UserIdentity.php(38): CActiveRecord->__get('username')
#2 C:\projeto-yii\protected\models\LoginForm.php(55): UserIdentity->authenticate()
#3 C:\Yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('senha', Array)
#4 C:\Yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'senha')
#5 C:\Yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\projeto-yii\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\projeto-yii\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 16:38:23 [error] [exception.CException] CException: Property "UserIdentity._id" is not defined. in C:\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\projeto-yii\protected\components\UserIdentity.php(59): CComponent->__get('_id')
#1 C:\Yii\framework\web\auth\CWebUser.php(231): UserIdentity->getId()
#2 C:\projeto-yii\protected\models\LoginForm.php(74): CWebUser->login(Object(UserIdentity), 0)
#3 C:\projeto-yii\protected\controllers\SiteController.php(94): LoginForm->login()
#4 C:\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\Yii\framework\web\CWebApplication.php(282): CController->run('login')
#9 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\projeto-yii\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost:3000/index.php?r=site/login
---
2022/08/04 17:22:16 [error] [exception.ParseError] ParseError: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in C:\projeto-yii\protected\views\post\_form.php:18
Stack trace:
#0 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#1 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#2 C:\projeto-yii\protected\views\post\create.php(19): CController->renderPartial('_form', Array)
#3 C:\Yii\framework\web\CBaseController.php(126): require('C:\\projeto-yii\\...')
#4 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#5 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#6 C:\Yii\framework\web\CController.php(785): CController->renderPartial('create', Array, true)
#7 C:\projeto-yii\protected\controllers\PostController.php(27): CController->render('create', Array)
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=post/create
HTTP_REFERER=http://localhost:3000/index.php?r=post
---
2022/08/04 17:23:58 [error] [exception.ParseError] ParseError: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in C:\projeto-yii\protected\views\post\_form.php:18
Stack trace:
#0 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#1 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#2 C:\projeto-yii\protected\views\post\create.php(19): CController->renderPartial('_form', Array)
#3 C:\Yii\framework\web\CBaseController.php(126): require('C:\\projeto-yii\\...')
#4 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#5 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#6 C:\Yii\framework\web\CController.php(785): CController->renderPartial('create', Array, true)
#7 C:\projeto-yii\protected\controllers\PostController.php(27): CController->render('create', Array)
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=post/create
HTTP_REFERER=http://localhost:3000/index.php?r=post
---
2022/08/04 17:24:14 [error] [exception.ParseError] ParseError: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in C:\projeto-yii\protected\views\post\_form.php:18
Stack trace:
#0 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#1 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#2 C:\projeto-yii\protected\views\post\create.php(19): CController->renderPartial('_form', Array)
#3 C:\Yii\framework\web\CBaseController.php(126): require('C:\\projeto-yii\\...')
#4 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#5 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#6 C:\Yii\framework\web\CController.php(785): CController->renderPartial('create', Array, true)
#7 C:\projeto-yii\protected\controllers\PostController.php(27): CController->render('create', Array)
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=post/create
HTTP_REFERER=http://localhost:3000/index.php?r=post
---
2022/08/04 17:26:16 [error] [exception.ParseError] ParseError: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in C:\projeto-yii\protected\views\post\_form.php:18
Stack trace:
#0 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#1 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#2 C:\projeto-yii\protected\views\post\create.php(19): CController->renderPartial('_form', Array)
#3 C:\Yii\framework\web\CBaseController.php(126): require('C:\\projeto-yii\\...')
#4 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#5 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#6 C:\Yii\framework\web\CController.php(785): CController->renderPartial('create', Array, true)
#7 C:\projeto-yii\protected\controllers\PostController.php(27): CController->render('create', Array)
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=post/create
HTTP_REFERER=http://localhost:3000/index.php?r=post
---
2022/08/04 17:32:20 [error] [exception.Error] Error: Function name must be a string in C:\projeto-yii\protected\views\post\_form.php:18
Stack trace:
#0 C:\Yii\framework\web\CBaseController.php(126): require()
#1 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#2 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#3 C:\projeto-yii\protected\views\post\create.php(19): CController->renderPartial('_form', Array)
#4 C:\Yii\framework\web\CBaseController.php(126): require('C:\\projeto-yii\\...')
#5 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#6 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#7 C:\Yii\framework\web\CController.php(785): CController->renderPartial('create', Array, true)
#8 C:\projeto-yii\protected\controllers\PostController.php(27): CController->render('create', Array)
#9 C:\Yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#10 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\Yii\framework\web\CWebApplication.php(282): CController->run('create')
#14 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#15 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\projeto-yii\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=post/create
HTTP_REFERER=http://localhost:3000/index.php?r=post
---
2022/08/04 17:40:28 [error] [exception.ParseError] ParseError: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in C:\projeto-yii\protected\views\post\_form.php:18
Stack trace:
#0 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#1 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#2 C:\projeto-yii\protected\views\post\create.php(19): CController->renderPartial('_form', Array)
#3 C:\Yii\framework\web\CBaseController.php(126): require('C:\\projeto-yii\\...')
#4 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#5 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#6 C:\Yii\framework\web\CController.php(785): CController->renderPartial('create', Array, true)
#7 C:\projeto-yii\protected\controllers\PostController.php(27): CController->render('create', Array)
#8 C:\Yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#9 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#14 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\projeto-yii\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=post/create
HTTP_REFERER=http://localhost:3000/index.php?r=post
---
2022/08/04 17:50:25 [error] [php] include(Categoria.php): failed to open stream: No such file or directory (C:\Yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\projeto-yii\protected\views\post\_form.php(23): spl_autoload_call()
#1 C:\Yii\framework\web\CBaseController.php(126): require()
#2 C:\Yii\framework\web\CBaseController.php(95): PostController->renderInternal()
#3 C:\Yii\framework\web\CController.php(872): PostController->renderFile()
#4 C:\projeto-yii\protected\views\post\create.php(19): PostController->renderPartial()
#5 C:\Yii\framework\web\CBaseController.php(126): require()
#6 C:\Yii\framework\web\CBaseController.php(95): PostController->renderInternal()
#7 C:\Yii\framework\web\CController.php(872): PostController->renderFile()
#8 C:\Yii\framework\web\CController.php(785): PostController->renderPartial()
#9 C:\projeto-yii\protected\controllers\PostController.php(27): PostController->render()
#10 C:\Yii\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#11 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#12 C:\Yii\framework\web\CController.php(286): PostController->runAction()
#13 C:\Yii\framework\web\CController.php(265): PostController->runActionWithFilters()
#14 C:\Yii\framework\web\CWebApplication.php(282): PostController->run()
#15 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=post/create
in C:\projeto-yii\protected\views\post\_form.php (23)
in C:\projeto-yii\protected\views\post\create.php (19)
in C:\projeto-yii\protected\controllers\PostController.php (27)
=======
2022/08/03 14:34:09 [error] [php] Use of undefined constant senha - assumed 'senha' (this will throw an Error in a future version of PHP) (C:\projeto-yii\protected\views\usuario\_view.php:14)
Stack trace:
#0 C:\Yii\framework\zii\widgets\CListView.php(291): UsuarioController->renderFile()
#1 C:\Yii\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#2 unknown(0): CListView->renderSection()
#3 C:\Yii\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback()
#4 C:\Yii\framework\zii\widgets\CBaseListView.php(149): CListView->renderContent()
#5 C:\Yii\framework\web\CBaseController.php(182): CListView->run()
#6 C:\projeto-yii\protected\views\usuario\index.php(18): UsuarioController->widget()
#7 C:\Yii\framework\web\CBaseController.php(126): require()
#8 C:\Yii\framework\web\CBaseController.php(95): UsuarioController->renderInternal()
#9 C:\Yii\framework\web\CController.php(872): UsuarioController->renderFile()
#10 C:\Yii\framework\web\CController.php(785): UsuarioController->renderPartial()
#11 C:\projeto-yii\protected\controllers\UsuarioController.php(60): UsuarioController->render()
#12 C:\Yii\framework\web\actions\CInlineAction.php(49): UsuarioController->actionIndex()
#13 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#14 C:\Yii\framework\web\CController.php(286): UsuarioController->runAction()
#15 C:\Yii\framework\web\CController.php(265): UsuarioController->runActionWithFilters()
#16 C:\Yii\framework\web\CWebApplication.php(282): UsuarioController->run()
#17 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#19 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=usuario/index
in C:\projeto-yii\protected\views\usuario\_view.php (14)
in C:\projeto-yii\protected\views\usuario\index.php (18)
in C:\projeto-yii\protected\controllers\UsuarioController.php (60)
2022/08/03 14:34:23 [error] [php] Undefined variable: senha (C:\projeto-yii\protected\views\usuario\_view.php:14)
Stack trace:
#0 C:\Yii\framework\zii\widgets\CListView.php(291): UsuarioController->renderFile()
#1 C:\Yii\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#2 unknown(0): CListView->renderSection()
#3 C:\Yii\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback()
#4 C:\Yii\framework\zii\widgets\CBaseListView.php(149): CListView->renderContent()
#5 C:\Yii\framework\web\CBaseController.php(182): CListView->run()
#6 C:\projeto-yii\protected\views\usuario\index.php(18): UsuarioController->widget()
#7 C:\Yii\framework\web\CBaseController.php(126): require()
#8 C:\Yii\framework\web\CBaseController.php(95): UsuarioController->renderInternal()
#9 C:\Yii\framework\web\CController.php(872): UsuarioController->renderFile()
#10 C:\Yii\framework\web\CController.php(785): UsuarioController->renderPartial()
#11 C:\projeto-yii\protected\controllers\UsuarioController.php(60): UsuarioController->render()
#12 C:\Yii\framework\web\actions\CInlineAction.php(49): UsuarioController->actionIndex()
#13 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#14 C:\Yii\framework\web\CController.php(286): UsuarioController->runAction()
#15 C:\Yii\framework\web\CController.php(265): UsuarioController->runActionWithFilters()
#16 C:\Yii\framework\web\CWebApplication.php(282): UsuarioController->run()
#17 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#19 C:\projeto-yii\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=usuario/index
in C:\projeto-yii\protected\views\usuario\_view.php (14)
in C:\projeto-yii\protected\views\usuario\index.php (18)
in C:\projeto-yii\protected\controllers\UsuarioController.php (60)
2022/08/03 14:37:19 [error] [exception.Error] Error: Call to a member function getAttributeLabel() on string in C:\projeto-yii\protected\views\usuario\_view.php:13
Stack trace:
#0 C:\Yii\framework\web\CBaseController.php(130): require()
#1 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, false)
#2 C:\Yii\framework\zii\widgets\CListView.php(291): CBaseController->renderFile('C:\\projeto-yii\\...', Array)
#3 C:\Yii\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#4 [internal function]: CBaseListView->renderSection(Array)
#5 C:\Yii\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback('/{(\\w+)}/', Array, '{summary}\n{sort...')
#6 C:\Yii\framework\zii\widgets\CBaseListView.php(149): CBaseListView->renderContent()
#7 C:\Yii\framework\web\CBaseController.php(182): CBaseListView->run()
#8 C:\projeto-yii\protected\views\usuario\index.php(18): CBaseController->widget('zii.widgets.CLi...', Array)
#9 C:\Yii\framework\web\CBaseController.php(126): require('C:\\projeto-yii\\...')
#10 C:\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\projeto-yii\\...', Array, true)
#11 C:\Yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\projeto-yii\\...', Array, true)
#12 C:\Yii\framework\web\CController.php(785): CController->renderPartial('index', Array, true)
#13 C:\projeto-yii\protected\controllers\UsuarioController.php(60): CController->render('index', Array)
#14 C:\Yii\framework\web\actions\CInlineAction.php(49): UsuarioController->actionIndex()
#15 C:\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#17 C:\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\Yii\framework\web\CWebApplication.php(282): CController->run('index')
#19 C:\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usuario/index')
#20 C:\Yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#21 C:\projeto-yii\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php?r=usuario/index
HTTP_REFERER=http://localhost:3000/index.php?r=usuario/view&id=2
---
>>>>>>> main
